tango_simlib/tests/DishElementMaster_SIMDD.json

```diff
@@ -190,6 +190,70 @@
           "quantity_simulation_type": "ConstantQuantity"
         }
       }
+    },
+    {
+      "basicAttributeData": {
```

Comment:
It is. I have removed it. I had thought that the we needed to specify the `quantity_simulation_type` parameter to be able to create the quantities, however the `PopulateModelQuantities` class handles that. It creates the `ConstantQuantity` types as default quantity when it has not been specified.